packages/components/src/DateTimePickers/InputDateTimePicker/__snapshots__/InputDateTimePicker.test.js.snap

```diff
@@ -36,7 +36,7 @@ exports[`InputDateTimePicker render should render 1`] = `
           onSubmit={[Function]}
           selection={
             Object {
-              "date": 2017-04-03T22:00:00.000Z,
+              "date": 2017-04-03T21:00:00.000Z,
```

Comment:
Seems that you run your tests without the `test` script.
Please use `yarn test` that sets the timezone.